requirements.txt

```diff
@@ -4,7 +4,9 @@ alembic>=0.8.10 # MIT
 eventlet==0.24.1
 flake8>= 3.7.7
 Flask>=1.0.3
+flask-keystone==0.2
 flask-migrate>=2.5.2
+flask-oslolog==0.1
```

Comment:
@larsks Do you know if there are policies as to which libraries we should use? Both flask-keystone and flask-oslolog are actually Rackspace projects; are there worries that they'd end up unsupported?

Reply:
@ljmcgann I don't think you need these here anymore - we don't use them, correct?

Reply:
Little things like this are so easy to forget

Reply:
Haha, yeah, I've definitely done the same